src/modules/components/dao/daoAvatar/daoAvatar.tsx

```diff
@@ -1,12 +1,17 @@
 import classNames from 'classnames';
 import type React from 'react';
-import { Avatar, type IAvatarProps } from '../../../../core';
+import { Avatar, type AvatarSize, type IAvatarProps } from '../../../../core';
 
 export interface IDaoAvatarProps extends Omit<IAvatarProps, 'fallback'> {
     /**
      * Name of the DAO
      */
     name?: string;
+    /**
+     * The size of the avatar.
+     * @default lg
+     */
+    size?: AvatarSize;
```

Comment:
The `size` property is already set on the `IAvatarProps` interface, we don't need to specify it here as well

Reply:
I had added this after talking with @Fabricevladimir because the autodocs pick up the wrong default for Storybook.